.github/actions/deploy-ubuntu/action.yml

```diff
@@ -334,6 +339,7 @@ runs:
           export MAVEN_OPTS="-Xss2m -Xmx4g"
           export MAVEN_OPTIONS="clean $MAVEN_PHASE -B -U -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.http.retryHandler.count=3 -Djavacpp.platform=$CI_DEPLOY_PLATFORM -Djavacpp.platform.extension=$EXT $MAVEN_OPTIONS $CI_DEPLOY_OPTIONS"
           for i in {1..5}; do STATUS=0; mvn $MAVEN_OPTIONS -pl .,$CI_DEPLOY_MODULE && break || STATUS=$? && sleep 60; done; (exit $STATUS)
+          mvn clean -B -e -pl $CI_DEPLOY_MODULE # Free some disk space
```

Comment:
That might start downloading dependencies for other platforms... We can put clean after install too like "mvn clean install clean"

Reply:
Ok. "install clean" is probably enough. Can the tree be dirty before first run ?